### ISSUE TITLE
Dashboard: Copy Paste rows between dashboards

### ISSUE THREAD

**Issue by nmcc1212** [grafana/grafana#23762]
**What would you like to be added**:
A copy and paste feature for rows similar to the existing one for panels

**Why is this needed**:

To allow many panels to quickly be transferred between dashboards

**Comment by nmcc1212** [comment]
Is there anything I can provide to help, as it says awaiting more evidence

**Comment by torkelo** [comment]
you can do this via the panel menu, More -> copy 

then in the Add panel view choose "Paste copied panel"

**Comment by torkelo** [comment]
ah, this is for rows.

**Comment by nmcc1212** [comment]
yes

**Comment by nmcc1212** [comment]
I am asking for the same functionality just with rows

**Comment by nmcc1212** [comment]
this isssue got burried so restarted it in #25955 25955

**Comment by ksquaredkey** [comment]
Why is this closed?  This has been asked for by many users since Issue #1004.  AFAIK, we still cannot copy/paste Rows like we can with panels.  And suggesting Jsonnet/Grafananet is going completely in the opposite direction of the level of user sophistication that I need this function for.

**Comment by Teressss** [comment]
yeah, I would so much want to have simple option to copy/duplicate row

**Comment by songgiwon** [comment]
please,,, I would want to have simple option to copy/duplicate 'ROW'

**Comment by mdvictor** [comment]
I think this is relevant and we still have an eye on fixing this, although I don't have a timeline for it.

**Comment by nmcc1212** [comment]
still relevant

**Comment by gianfun** [comment]
still relevant

**Comment by thorstenvogel** [comment]
still relevant 👍

**Comment by x64alex** [comment]
still relevant

**Comment by cygnuson** [comment]
Bump!

**Comment by skufee** [comment]
bump

**Comment by r-furlan** [comment]
Bump!

**Comment by deeTEEcee** [comment]
bump

**Comment by yaelleC** [comment]
This is now possible using the [new editing experience](https://grafana.com/whats-new/2026-01-14-dynamic-dashboards-in-public-preview/) 🎉 

<img width="372" height="251" alt="Image" src="https://github.com/user-attachments/assets/5a4782df-c0b6-4c52-aec5-0ce6056ee3af" />
